src/Microsoft.Data.SqlClient/tests/ManualTests/AlwaysEncrypted/ApiShould.cs

```diff
@@ -31,12 +31,13 @@ public ApiShould(SQLSetupStrategyCertStoreProvider fixture)
             tableName = fixture.ApiTestTable.Name;
         }
 
-        [ConditionalTheory(typeof(DataTestUtility), nameof(DataTestUtility.AreConnStringsSetup))]
-        [InlineData(true)]
-        [InlineData(false)]
-        public void TestSqlTransactionCommitRollbackWithTransparentInsert(bool isCommitted)
+        [ConditionalTheory(typeof(DataTestUtility), nameof(DataTestUtility.AreConnStringSetupForAE))]
+        [ClassData(typeof(AEConnectionStringProviderWithBooleanVariable))]
+        public void TestSqlTransactionCommitRollbackWithTransparentInsert(string connection, bool isPermitted)
```

Comment:
is there a reason `isCommitted` is changed to `isPermitted`?

Reply:
Reverted